lte/gateway/deploy/roles/dev_common/defaults/main.yml

```diff
@@ -1,7 +1,4 @@
 tmp_directory: /var/tmp
 codegen_root: /var/tmp/codegen
-mvn_version: 3.5.4
```

Comment:
If someone stumbles over this at a later point: removing these variables is basically unrelated to the intended change of this PR. This is just clean-up of unused variables.